clang/lib/CIR/Dialect/Transforms/CIRSimplify.cpp

```diff
@@ -107,6 +111,91 @@ struct RemoveTrivialTry : public OpRewritePattern<TryOp> {
   }
 };
 
+/// Simplify suitable ternary operations into select operations.
+///
+/// Only those ternary operations that meet the following criteria can be
```

Comment:
Remove this whole paragraph from the comments: there's nothing cost or side effect related in the actual implemented transformation. The rest looks pretty good though, thanks for detailed comment.

Reply:
Updated.